### PR TITLE
Add a bit of information to example

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

**Review by ISSOtm — APPROVED** [review]
While we're here, please:

- Remove spaces before the colons
- Turn the postfix binary into prefix (`00b` → `%00`)

I know it's unrelated to the rest of this change, but the results matter most.

**Comment by AlvaroBurnett** [comment]
I think you mean the opposite: I used prefix binary and you want postfix. Correct?

**Comment by ISSOtm** [comment]
No, we want prefix because that's RGBDS syntax, which has de-facto become the standard around here.

**Comment by AlvaroBurnett** [comment]
ok, I think this is ready to be merged.

**Comment by avivace** [comment]
I don't think we ever decided on the `%` prefix. https://github.com/gbdev/pandocs/wiki/Document-Style

**Comment by AlvaroBurnett** [comment]
Nintendo uses the postfix 'b' format in their programming manual.

**Comment by avivace** [comment]
I think this is related: https://github.com/gbdev/pandocs/issues/11

File: content/Video_Display.md [hunk 1]
```diff
@@ -479,10 +479,11 @@ etc.`
...
-Bit 0 the rightmost. For example : let's say you have \$57 \$36 (in
-this order in memory). To obtain the color index for the leftmost pixel,
-you take bit 7 of both bytes : 0, and 0. Thus the index is 00b = 0. For
-the second pixel, repeat with bit 6 : 1, and 0. Thus the index is 01b =
+Bit 0 the rightmost. For example: let's say you have \$57 \$36 (in
+this order in memory), which in binary are %01010111 and %00110110.
+To obtain the color index for the leftmost pixel,
+you take bit 7 of both bytes: 0, and 0. Thus the index is %00 = 0. For
+the second pixel, repeat with bit 6: 1, and 0. Thus the index is %01 =
 1 (remember to flip the order of the bits !). If you repeat the
```
Comment (ISSOtm) [inline content/Video_Display.md:487]: No spaces before exclamation marks, either.

**Review by ISSOtm — APPROVED** [review]
Alright, overall this is good. We'll need to squash some of these commits (though preferably not the whole PR).

**Comment by AlvaroBurnett** [comment]
Damnit I did something wrong with the squash...

**Pull request closed without merging**